src/development/syscall.md

```diff
@@ -103,14 +103,14 @@ The steps from the overview are elaborated on here.
       return value of the command is the maximum water level.
     - _What is the current water level?_ This will require two steps. First,
       there needs to be a subscribe call where the application can setup a
-      callback function. The kernel will call this when the water level value
+      upcall function. The kernel will call this when the water level value
```

Comment:
```suggestion
      upcall function. The kernel will call this when the water level value
```

an upcall -- github being dumb and won't let me change the line above this here :/